src/main/java/org/traccar/model/ExtendedModel.java

```diff
@@ -154,4 +154,7 @@ public long getLong(String key) {
         }
     }
 
+    public Object removeAttribute(String key) {
```

Comment:
Should we equivalent for each type? Like `removeString`, `removeLong` etc? For consistency

Reply:
I don't think so. 
The purpose and intention of a user of that method is to remove the parameter and have its parameter as an optional "side effect" rather than get the nice typed value (which is covered by the existing methods) and have the removal as a side effect.

Reply:
If you look at the pull request, you'll noticed that we almost always use the result. So I would argue it's not a side effect, but the main purpose.

Reply:
For Java collections it's pretty standard to return value or remove, so I don't see the name as a problem.